src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Input validation purpose
+ */
+
+public class Contact {
+
+    public final String value;
+    public boolean isPrivate;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.